docs/glossary.rst

```diff
@@ -0,0 +1,66 @@
+===============================================================================
+XBRL Glossary
+===============================================================================
+
+Axis
+^^^^
+
+Axes are defined in the Taxonomy, and are used as a part of the
+Context to identify a Fact beyond the default
+entity, and Period. Axes can be either typed or explicit. An
+explicit Axis will have all of its allowable values defined in the taxonomy. A
+typed Axis will instead allow a filer to input their own values for the Axis.
+
+Concept
+^^^^^^^
+
+Concepts are defined in the Taxonomy, and describe what sort of Facts can be
+reported in a Filing. They provide a written description, and other metadata that
+the taxonomy creator chooses to include. Not all Concepts will have Facts they are
+directly associated with. This is because Concepts can also be used as a part of a
+Concept tree to define relationships with other Concepts. The root of such a tree
+is a Link Role.
+
+Context
+^^^^^^^
+
+A context provides important information for identifying/interpreting a
+Fact. Every context contains both an entity identifier, and a
+Period. A context can also contain user defined Axes.
+
+Fact
+^^^^
+
+An XBRL fact is considered to be an atomic unit of data. It contains a single value,
+with a name that corresponds to a Concept, and a reference to a
+Context.
+
+Filing
```

Comment:
What does this mean in the context of the FERC data? Each filing is all of the data reported by a single utility (respondent / entity) in a single year (or other reporting period), so it contains information that will end up being turned into one or more rows in many (or even all) of the different tables.

Can we say anything general and understandable about how the DB is a concatenation of many individual filings?

Reply:
Added note